tests/resources/functional/snowflake/test_tokens_parameter/test_tokens_parameter_4.sql

```diff
@@ -0,0 +1,5 @@
+-- snowflake sql:
+select &col_name from TEST_USER.EMP_TBL;
+
+-- databricks sql:
+select $col_name from TEST_USER.EMP_TBL;
```

Comment:
where do we use this syntax? please add references to documentation.

Reply:
making necessary changes